check-osp/osp_checks.py

```diff
@@ -38,6 +38,27 @@ def check_vm_count(system, warn=10, crit=15, **kwargs):
     sys.exit(3)
 
 
+def check_image_count(system, warn=10, crit=15, **kwargs):
+    """ Check overall Image count. """
+    warn = int(warn)
+    crit = int(crit)
+    template_count = len(system.list_templates())
+    # determine ok, warning, critical, unknown state
+    if template_count < warn:
+        print("Ok: Image count is less than {}. Image Count = {}".format(warn, template_count))
+        sys.exit(0)
+    elif template_count >= warn and template_count <= crit:
+        print("Warning: Image count is >=  {} & <= {}. Image Count = {}"
+            .format(warn, crit, template_count))
+        sys.exit(1)
+    elif template_count > crit:
+        print("Critical: Image count is greate than crit. Image Count = {}".format(crit, template_count))
```

Comment:
One `{}` missing here. should be 
`print("Critical: Image count is greater than critical {}. Image Count = {}".format(crit, template_count))`

Reply:
Oops :smile: